charts/prometheus-nginx-exporter/README.md

```diff
@@ -0,0 +1,65 @@
+# prometheus-nginx-exporter
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![AppVersion: 0.11.0](https://img.shields.io/badge/AppVersion-0.11.0-informational?style=flat-square)
+
+A Helm chart for the Prometheus NGINX Exporter
+
+**Homepage:** <https://github.com/nginxinc/nginx-prometheus-exporter>
+
+## Source Code
+
+* <https://github.com/nginxinc/nginx-prometheus-exporter>
+* <https://github.com/prometheus-community/helm-charts>
+
+## Values
```

Comment:
We don't add the values in the readme again in this repo. 
Pleas ad needed documentation to the values file.